.localization-config

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I don't think this file is needed. @nokura do you know?

Reply:
I believe we don't need this file, but will double check with Loc engineer for sure. Will get back.